src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,32 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+echo Remove Https configuration
+
+net session > NUL
+if NOT [%_ErrorLevel%]==[0] (
```

Comment:
When this script runs on my machine (in an elevated CMD window), I get this error, and the https ports are not removed:
```
Remove Https configuration
Error: this script must be called in an elevated window
```
I think the error is using an underscore in %_ErrorLevel%.  When I remove the underscore, it gets past this point.